jenkins-worker/Worker_Dockerfile

```diff
@@ -0,0 +1,78 @@
+  # Based off of evarga/jenkins-slave
+# This Dockerfile is used to build an image containing basic stuff to be used as a Jenkins slave build node.
+FROM ubuntu:trusty
+
+# This was the original, trying to make it match the frontend version
+#FROM ubuntu:14.04
+# Original mainainer: Matt Shanker
+MAINTAINER engineering@socrata.com
+
+# Add lines in limits.conf to make the ulimit higher, default of 1024 is from the stone age
+RUN echo '*  	hard	    nofile  	525488' >> /etc/security/limits.conf
+RUN echo '*  	soft	    nofile  	525488' >> /etc/security/limits.conf
+
+# Add lines to the locale related files
+RUN echo "LC_ALL=en_US.UTF-8" >> /etc/environment
+RUN echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
+RUN echo "LANG=en_US.UTF-8" > /etc/locale.conf
```

Comment:
The other two commands are adding to the files in question, but this one is overwriting the file. 🤔

Reply:
I believe that overwrite is required.